### PR TITLE
Add ayah menu item to allow playing selected verses in a loop

### PR DESCRIPTION
**Pull request by zukhan**
### Summary:

This change allows the user the ability to easily repeat selected verse(s). Currently: 

1. If you want to repeat multiple selected verses, you have to select the verses, click on "Play selected ayahs", navigate to audio options on the bottom right, click on "loop" in "Play set of verses" and then click on the Play button. 

2. If you want to repeat only one selected verse, you have to select multiple verses first, click on "Play selected ayahs", navigate to audio options on the bottom right, change the end verse to be the same as the start verse, then click on "loop" in "Play set of verses", then click on the Play button.

This change allows repeating one or more verses with a single menu item press after selecting the verse(s).

### Screenshots

**Single Verse Selected**

single-verse-selected

**Multiple Verses Selected**

multiple-verses-selected

### CODE REVIEW

File: Sources/Localization/Resources/ar.lproj/Android.strings [hunk 1]
```diff
@@ -181,6 +181,7 @@
 "prefs_use_arabic_summary_on" = "سيتم تحويل التطبيق إلى اللغة العربية";
 "prefs_use_arabic_title" = "الوضع العربي";
 "play" = "تشغيل";
+"repeat" = "كرر";
```
Comment (mohamede1945) [inline Sources/Localization/Resources/ar.lproj/Android.strings:184]: The Android.strings files are autogenerated from the Android repo, let's not use it. Let's use Localizable.strings for strings dedicated to iOS only, please.

File: Sources/Localization/Resources/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -121,7 +121,9 @@
 "ayah.menu.play-end-juz" = "to the end of Juz'";
 "ayah.menu.play-end-surah" = "to the end of Surah";
 "ayah.menu.play-end-page" = "to the end of Page";
-"ayah.menu.play-selected-verses" = "selected verses";
+"ayah.menu.play-selected-verses-once" = "selected verses (once)";
```
Comment (mohamede1945) [inline Sources/Localization/Resources/en.lproj/Localizable.strings:124]: I don't think "Once" adds too much value. It also increases the width of the Ayah menu, not a big deal, but something to note.

**Review by mohamede1945 — CHANGES_REQUESTED** [review]
Jazak Allah khyran for the great contribution!

File: Sources/Localization/Resources/ar.lproj/Android.strings (hunk 1 above)
Comment (zukhan) [reply Sources/Localization/Resources/ar.lproj/Android.strings:184]: Good to know, thanks!

Comment (zukhan) [reply Sources/Localization/Resources/ar.lproj/Android.strings]: Fixed.

File: Sources/Localization/Resources/en.lproj/Localizable.strings (hunk 2 above)
Comment (zukhan) [reply Sources/Localization/Resources/en.lproj/Localizable.strings]: Removed it.

**Review by mohamede1945 — APPROVED** [review]

**Comment by mohamede1945** [comment]
Jazak Allah khyran, Zubair!

**Pull request merged**